max-min-array.js

```diff
@@ -0,0 +1,21 @@
+/* Your task is to make two functions (max and min, or maximum and minimum, etc., depending on the language) 
+that receive a list of integers as input and return, respectively, the largest and lowest number in that list.
+Examples (Input -> Output)
+
+* [4,6,2,1,9,63,-134,566]         -> max = 566, min = -134
+* [-52, 56, 30, 29, -54, 0, -110] -> min = -110, max = 56
+* [42, 54, 65, 87, 0]             -> min = 0, max = 87
+* [5]                             -> min = 5, max = 5
+ */
+
+
+
+
+let max = function rmin(){
+ return Math.min(...list);
+}
+let min = function rmax(){
+  return Math.max(...list);
+}
```

Comment:
Since you don't ever re-assign `max` / `min` they can be `const`. However, the way the functions are set up is likely to cause unexpected results / bugs because `list` isn't scoped to the function. 
```
const list = [1,4,7,4,6,8];

max(...list)
// 8 

min(...list)
// 1

min([2,4,6])
// 1
// This still returns 1 because of the scope.
```

You can also simplify to a one-liner without creating the references of `min` / `max` and by using arrow functions:
  ```suggestion
const rmin = numbers => Math.min(...numbers);
const rmax = numbers => Math.max(...numbers);
rmax(list); // 8
rmin([3,7,6]); // 3
```